lib/perl/Genome/Model/Tools/Validation/ProcessSomaticValidation.pm

```diff
@@ -619,6 +635,11 @@ sub execute {
   my $ref_seq_build = Genome::Model::Build->get($ref_seq_build_id);
   my $ref_seq_fasta = $ref_seq_build->full_consensus_path('fa');
   my $annotation_build_name = $model->annotation_build->name;
+  if(defined $self->reference_transcripts){
+      print STDERR "Model's annotation build overriden. Using " . $self->reference_transcripts . "\n";
```

Comment:
I think if this were to come up in the future this should be suggested as a separate PR.  I think the argument would be that it would be better to replace `print` with `*_message` but local consistency suggests that should be done in bulk and do to the scope of that it should be a separate PR.